### PR TITLE
Dark mode & Logic Additions

### PR DESCRIPTION
**Pull request by PixelPoncho**
Have fun finishing the rest of this

### CODE REVIEW

File: cgi/templates/admin/editPick.ejs [hunk 1]
```diff
@@ -1,18 +1,25 @@
 <!DOCTYPE html>
 <head>
     <%- include('../head_shit.ejs', {title: 'EWMA | Admin', request: request, cacheBuster: cacheBuster}); %>
+    <link href='https://fonts.googleapis.com/css?family=Didact Gothic' rel='stylesheet'>
```
Comment (bufutda) [inline cgi/templates/admin/editPick.ejs]: HTML attributes should have double quotes

File: cgi/templates/admin/editPick.ejs [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
 </head>
+
+<style>
```
Comment (bufutda) [inline cgi/templates/admin/editPick.ejs]: Make an external stylesheet

File: cgi/templates/admin/user.ejs [hunk 3]
```diff
@@ -1,63 +1,85 @@
...
-                        <tr>
-                            <td>
-                                <label for="email">Email</label>
-                            </td>
-                            <td><input type="email" name="email" disabled="disabled" value="<%= user.attributes.email %>"></td>
-                        </tr>
-                        <tr>
-                            <td><label for="permissions">Permissions</label></td>
-                            <td class="monospace">
+            <div class="user-info">
```
Comment (bufutda) [inline cgi/templates/admin/user.ejs]: indenting pls

File: cgi/views/admin/deletepermission.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
+/**
+ * Add new permission to the database
+ * @author Mitchell Sawatzky
```
Comment (bufutda) [inline cgi/views/admin/deletepermission.js]: This is not the author of this file

File: cgi/views/admin/newpermission.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+/**
+ * Add new permission to the database
+ * @author Mitchell Sawatzky
```
Comment (bufutda) [inline cgi/views/admin/newpermission.js]: This too

File: cgi/views/admin/newpermission.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    request.db.do(`INSERT INTO permission_schema (perm, description) VALUES (?, ?)`, [
+        request.post.name,
+        request.post.description
+    ], (err) => {
+        if (err) {
+            throw err;
+        }
+        redirection.found(request, '/admin/permissions', cb);
+    });
+});
```
Comment (bufutda) [inline cgi/views/admin/newpermission.js]: Missing newline

File: cgi/views/admin/user.js [hunk 7]
```diff
@@ -13,37 +13,61 @@ module.exports.name = 'user management';
...
+                        // tells you the permissions that the user does not have
+                        request.db.do('SELECT perm FROM permission_schema', (err, rows) => {
+                            if (err) {
+                                throw err;
+                            }
+
+                            let existingPerms = [];
+                            for (let row of rows) {
+                                if (!u.permissions.has(row.perm)) {
+                                    existingPerms.push(row.perm)
```
Comment (bufutda) [inline cgi/views/admin/user.js]: missing semicolon

File: db/dump_ewma.sql [hunk 8]
```diff
@@ -0,0 +1,335 @@
+-- MySQL dump 10.13  Distrib 5.7.21, for Linux (x86_64)
```
Comment (bufutda) [inline db/dump_ewma.sql]: This file should not be committed

File: static/css/edit.sass [hunk 9]
```diff
@@ -1,9 +1,64 @@
...
-        text-align: right
-
-    td:nth-child(2), td:nth-child(4)
-        text-align: left
+body
+    background-color: #23272A
+
+    #site.movie-picks
+        #content
+            padding: 50px 50px 50px 50px
```
Comment (bufutda) [inline static/css/edit.sass]: padding: 50px

File: static/css/edit.sass [hunk 10]
```diff
@@ -1,9 +1,64 @@
...
+            align: center
+            max-width: 69%
+            padding: 20px 50px 0px 50px
```
Comment (bufutda) [inline static/css/edit.sass]: duplicate padding

File: static/css/edit.sass [hunk 11]
```diff
@@ -1,9 +1,64 @@
...
+            .movie-select
+                font-family: 'Didact Gothic'
+                font-size: 16px
+                padding-inline-start: 15px
+                width: 70%
+                padding-top: 10px
+                padding-bottom: 10px
+                border-radius: 20px
+                border: 1px inset #00000047
+                background-color: #52555c
```
Comment (bufutda) [inline static/css/edit.sass]: uppercase color

File: static/css/edit.sass [hunk 12]
```diff
@@ -1,9 +1,64 @@
...
+                color: #FFF
+                outline: none
+
+                option
+                    &:disabled
```
Comment (bufutda) [inline static/css/edit.sass]: this can be combined with just option

File: static/css/edit.sass [hunk 13]
```diff
@@ -1,9 +1,64 @@
...
+            .button
+                transform: translateX(-10px)
+                font-family: 'Didact Gothic'
+                font-size: 13px
+                margin-block-start: 25px
+                padding: 10px 20px 11px
+                font-weight: bold
+                border-radius: 100px
+                color: #23272A
+                background-color: #73d5ff
```
Comment (bufutda) [inline static/css/edit.sass]: uppercase color

File: static/css/headers.sass [hunk 14]
```diff
@@ -13,12 +15,14 @@
     table-layout: fixed
     padding-left: 180px
     text-align: center
+    a:
+        border: 1px solid #FFF
     a, ul
         text-decoration: none
-        font-variant: small-caps
+        // font-variant: small-caps
```
Comment (bufutda) [inline static/css/headers.sass]: remove that

File: static/css/index.sass [hunk 15]
```diff
@@ -1,3 +1,15 @@
...
     margin-top: 20px
+    color: #73d5ff
+    width: 400px
+    margin: 50px auto
+    padding: 20px
+    border-radius: 20px
+    background: #2c3037
+    width: 75%
+    box-shadow: #434444
+    border: 2px groove #99AAB5
```
Comment (bufutda) [inline static/css/index.sass]: missing newline

File: static/css/manage-user.sass [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+        border-radius: 10px
+
+    thead tr
+        background: #2c3037
+
+    tr:nth-child(2n)
+        background: #2c3037
+
+    a.user-email
+        text-decoration: underline !important
```
Comment (bufutda) [inline static/css/manage-user.sass]: no important pls

File: static/css/newaccount.sass [hunk 17]
```diff
@@ -1,11 +1,75 @@
...
+        .formPart.email
+            margin-block-end: 10px
+            width: 75%
+        .user-password
+            margin-block-end: 10px
+            .formPart.initial-pwd,
+            .formPart.retype-pwd,
+                width: 75%
+
+
```
Comment (bufutda) [inline static/css/newaccount.sass]: extra spaces

File: static/js/edgeCases.js [hunk 18]
```diff
@@ -1,23 +0,0 @@
-function getInternetExplorerVersion()
```
Comment (bufutda) [inline static/js/edgeCases.js]: this should not have been removed

File: cgi/views/admin/deletepermission.js (hunk 4 above)
Comment (PixelPoncho) [reply cgi/views/admin/deletepermission.js]: But it is

File: cgi/views/admin/newpermission.js (hunk 5 above)
Comment (PixelPoncho) [reply cgi/views/admin/newpermission.js]: but it is

**Comment by PixelPoncho** [comment]
Check out the updated approach in PR #10

**Pull request closed without merging**